fossology/fossology
# feat(decisions): auto deactivate copyrights

## Description

Automatically deactivate the copyright statement if the user applies the license decision to irrelevant.  

## How to test

- Upload a package.
- Apply irrelevant decisions. from file level as well as folder level and check the copyrights.
- As this changes may affect the clearing decision part, please test every clearing part.
